### PR TITLE
fix: Adding id as prop to default and vault flyout menus

### LINKED ISSUE #1288: Check if we can set`id`s to `default` and `vault` flyout menus, since we add it as a prop
Should we also set `id`s to `default` and `vault` flyout menus, since we add it as a prop?

_Originally posted by @birtony in https://github.com/trustbloc/wallet/pull/1285#discussion_r763380043_

### PR DESCRIPTION
**Pull request by talwinder50**
closes #1288

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: cmd/wallet-web/src/pages/Credentials.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <!-- Mobile Credentials Layout -->
     <div v-if="breakpoints.xs || breakpoints.sm" class="flex flex-col justify-start w-screen">
       <div class="bg-neutrals-white border border-neutrals-chatelle">
-        <flyout-menu />
+        <flyout-menu id="defaultMobileFlyoutMenu" />
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Credentials.vue:12]: Can we give it more meaningful name? Something like `credentials-flyout-menu-mobile`

File: cmd/wallet-web/src/pages/Credentials.vue [hunk 2]
```diff
@@ -29,7 +29,7 @@
           border border-neutrals-chatelle
         "
       >
-        <flyout-menu />
+        <flyout-menu id="defaultFlyoutMenu" />
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Credentials.vue:32]: And `credentials-flyout-menu-desktop` here?

File: cmd/wallet-web/src/pages/Vaults.vue [hunk 3]
```diff
@@ -24,7 +24,7 @@
         :vault-id="vault.id"
       >
         <slot v-if="vault.name == 'Default Vault'">
-          <flyout-menu type="vault">
+          <flyout-menu type="vault" id="defaultVaultFlyoutMenu">
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Vaults.vue:27]: And `vaults-flyout-menu-default`

File: cmd/wallet-web/src/pages/Vaults.vue [hunk 4]
```diff
@@ -24,7 +24,7 @@
...
             <flyout-menu-list>
               <flyout-menu-button
                 id="renameVault"
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Vaults.vue:30]: Ohhh I just noticed we still have these hard-coded `id`s for `flyout-menu-button`s, we should not do that either

**Review by birtony — COMMENTED** [review]
Maybe you can come up with clearer `id`s, just wanted to say we should probably try to standardize these somehow and try to make them explicit

File: cmd/wallet-web/src/pages/Vaults.vue (hunk 4 above)
Comment (talwinder50) [reply cmd/wallet-web/src/pages/Vaults.vue:30]: As we discussed these flyout menu are rendered one at a time , so these ids should not be a problem

**Review by birtony — APPROVED** [review]

**Pull request merged**